ERP-BACKEND/app/routers/health.py

```
@@ -0,0 +1,378 @@
+            db.execute(text("SELECT 1"))
+            is_sqlite = "sqlite" in str(db.bind.engine.dialect.name).lower()
+            
+            if not is_sqlite:
+                # PostgreSQL-specific check
+                result = db.execute(text("SELECT pg_is_in_recovery()"))
+                is_replica = result.fetchone()[0]
+                checks["read_write"] = {"status": "ok", "is_replica": is_replica}
+            else:
+                # SQLite just confirms it can read/write
+                checks["read_write"] = {"status": "ok", "is_sqlite": True}
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)